app/types.ts

```diff
@@ -35,7 +36,7 @@ export type GalleryEntity = {
   event?: EntityID;
 };
 export type EncodedToken = string;
-export type DecodedToken = {
+export type DecodedToken = JwtPayload & {
```

Comment:
I'm not sure if I see the point of extending this off of `JwtPayload`, but okay. `exp` is a field there, so you could remove it here to avoid dups, but no biggy.

Reply:
Well yes but all properties in `JwtPayload` are optional. Extending this just means the type is more correct.